docs/Deployment/BackupRestore.md

```diff
@@ -0,0 +1,324 @@
+Removing generated_bitcoin_rtl_1               ... done
+Removing nginx                                 ... done
+Removing network generated_default
+
+ℹ️  Archiving files in /var/lib/docker…
+postgres.sql.gz
+volumes/generated_bitcoin_wallet_datadir/
+volumes/generated_bitcoin_wallet_datadir/_data/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/.walletlock
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/wallet.dat
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/db.log
+volumes/generated_btcpay_datadir/
+volumes/generated_btcpay_datadir/_data/
+volumes/generated_btcpay_datadir/_data/FastSynced
+volumes/generated_btcpay_datadir/_data/LocalStorage/
+volumes/generated_btcpay_datadir/_data/LocalStorage/tmp/
+volumes/generated_btcpay_datadir/_data/btcpay20220512.log
+volumes/generated_btcpay_datadir/_data/tmp/
+volumes/generated_btcpay_datadir/_data/host_id_rsa
+volumes/generated_btcpay_datadir/_data/Main/
+volumes/generated_btcpay_datadir/_data/Main/settings.config
+volumes/generated_btcpay_datadir/_data/host_id_rsa.pub
+volumes/generated_btcpay_datadir/_data/host_authorized_keys
+volumes/generated_btcpay_datadir/_data/key-47ec2b82-ed01-45ef-8a63-390470cc10e5.xml
+[...]
+volumes/generated_tor_torrcdir/
+volumes/generated_tor_torrcdir/_data/
+volumes/generated_tor_torrcdir/_data/torrc
+volumes/generated_tor_torrcdir/_data/torrc-2
+✅ Archive done.
+
+🔐 BTCPAY_BACKUP_PASSPHRASE is set, and the backup will be encrypted.
+✅ Encryption done.
+
+ℹ️  Restarting BTCPay Server …
+
+/root/btcpayserver-docker
+Creating network "generated_default" with the default driver
+Creating nginx                   ... done
+Creating generated_postgres_1    ... done
+Creating generated_bitcoin_rtl_1 ... done
+Creating tor                     ... done
+Creating btcpayserver_bitcoind   ... done
+Creating tor-gen                         ... done
+Creating generated_nbxplorer_1           ... done
+Creating btcpayserver_clightning_bitcoin ... done
+Creating generated_clightning_bitcoin_charge_1 ... done
+Creating generated_clightning_bitcoin_spark_1  ... done
+Creating generated_btcpayserver_1              ... done
+Creating generated_clightning_bitcoin_rest_1   ... done
+Creating nginx-gen                             ... done
+Creating letsencrypt-nginx-proxy-companion     ... done
+
+ℹ️  Cleaning up …
+
+✅ Backup done => /var/lib/docker/volumes/backup_datadir/_data/backup.tar.gz.gpg
+
+```
+
+## How does the BTCpay-restore.sh work? 
+
+It's very similar to the `btcpay-backup.sh` process but in reverse. 
+The `btcpay-restore.sh` script needs to be run with the path to your `backup.tar.gz` file. 
+
+First off, open a terminal and type the following as root.  
+Remember that if you set `BTCPAY_BACKUP_PASSPHRASE` on the backup, you also need to provide it for decryption :
+
+```
+cd "$BTCPAY_BASE_DIRECTORY"
+export BTCPAY_BACKUP_PASSPHRASE="tOpSeCrEt"
```

Comment:
I find it a bit inconsitent that for backup you specifiy the passphrase as an option and for restore you have to set an env variable. Couldn't we make the workflow a bit more consitant? Question towards @dennisreimann

Reply:
It works via env variable, I've updated the instructions.